test/evergreen.yml

```diff
@@ -923,6 +923,20 @@ tasks:
 
   # End of Python unit test tasks
 
+  - name: test-conf-dump
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+            cd test
+            ${test_env_vars|} ${python_binary|python} ../../test/wtperf/test_conf_dump.py 2>&1
```

Comment:
Not sure if these directory switchings are still necessary. Can we set the working_dir to  `wiredtiger/test/wtperf` on line 933 and call the script directly?